src/NServiceBus.Transport.Msmq.TransportTests/ConfigureMsmqTransportInfrastructure.cs

```diff
@@ -14,6 +14,7 @@ public TransportConfigurationResult Configure(SettingsHolder settings, Transport
     {
         var msmqTransportDefinition = new MsmqTransport();
         settingsHolder = settings;
+        settingsHolder.Set("NServiceBus.Transport.Msmq.MessageEnumeratorTimeout", TimeSpan.FromMilliseconds(10));
```

Comment:
SHould pass `MsmqSettings.MessageEnumeratorTimeout` instead of the magic number.

Reply:
Not following? (We want to use 10ms to get fast shutdown)